pkg/kds/context/full_sync_test.go

```diff
@@ -36,7 +36,7 @@ var _ = Describe("Full sync tests", func() {
 		for _, file := range files {
 			if strings.HasSuffix(file.Name(), ".input.yaml") {
 				zoneName := strings.TrimSuffix(file.Name(), ".input.yaml")
-				resourceStore := memory.NewStore()
+				resourceStore := store.NewPaginationStore(memory.NewStore())
```

Comment:
Do we need this for resource ordering? Or what's the motivation to wrap this?

Reply:
For resource ordering, we do `.List(..., Ordered)` and without this we won't returned